### ISSUE TITLE
Support Request: Roomba i6 *partially* supported?

### ISSUE THREAD

**Issue by DaveLinger** [bloomkd46/homebridge-iRobot#82]
### Describe Your Problem:

I have a Roomba i6. From my understanding, it's basically identical to the i7, except sold by Amazon and bundled with less extra bags and accessories (similarly I've heard the i8 is Costco's "verison" of the i7 except with lots of extra bags and accessories)

Anyway, The plugin works to the extent that it finds the roomba and I can start it - but when I switch to manual mode, I don't see any room switches, and if I try to start it, logs show it attempting to clean room "undefined", then fails.

The wiki doesn't seem to work (is it disabled?), so I can't really tell if I am missing something for multi-room support.

### Expected behavior:

I'm not 100% sure since the wiki is down, but I assume if I switch it to manual mode and try to start it, it should ask me which room?

### Logs:

```shell
[02/02/2022, 16:22:23] [iRobot] Set Roomba To Room-By-Room
[02/02/2022, 16:22:26] [iRobot] Set Roomba To Clean With args:undefined
```

### Config:

```shell
{
            "name": "iRobot",
            "email": "x",
            "password": "x",
            "roombas": [
                {
                    "autoConfig": true,
                    "info": {
                        "ver": 2
                    }
                }
            ],
            "manualDiscovery": false,
            "lowBattery": 20,
            "offAction": "stop:dock",
            "status": "cycle:clean",
            "eveStatus": "inverted:cycle:clean",
            "bin": "contact",
            "ignoreMultiRoomBin": true,
            "hideStuckSensor": false,
            "disableMultiRoom": false,
            "platform": "iRobotPlatform"
        }
```

### Node Version:

v16.13.1

### NPM Version:

8.3.0

### Homebridge Version:

1.4.0

### Plugin Version:

2.1.16

### Operating System:

Raspberian

**Comment by bloomkd46** [comment]
To make the rooms show up you must first run them from the iRobot app. This plug-in is adaptive so you will notice the rooms automatically show up after that.

Note: the rooms which show up as map 0 room <room ID number>. Could you please comment with a couple of the ID’s that showed up and what room they actually represented? I am trying to make a key.

**Comment by DaveLinger** [comment]
Thanks for the followup. When you say I must “first run” the rooms from the iRobot app before they’ll show up in the plug-in, what exactly do you mean?

I run custom jobs to clean specific rooms in the iRobot app all the time. Do I need to run them AFTER the plug-in has been installed?

**Comment by bloomkd46** [comment]
Yes, because the way this plug-in detects rooms is by looking at the last command the robot received which contains the rooms and their ID's

**Comment by DaveLinger** [comment]
Got it. Looks like i6 is fully supported then.

I ran it to vaccum the Hallway, Entryway, and Kitchen, and it created rooms 28, 31, and 33. I'll run some individually and map them for you.

**Comment by DaveLinger** [comment]
Weird behavior when I tried to stop/dock roomba:

`Set Roomba To stop and dock With args:undefined`

It stopped but didn't dock. After a minute, I checked the irobot app. It said saving map. After another minute, roomba started again, then stopped again. I tapped return home in the irobot app and it went home.

**Comment by DaveLinger** [comment]
Yeah, just tried this again. When I turn the roomba switch off in the home app, roomba just stops. It never docks, even though I have the stop action to "stop and dock" in my config. Roomba app said ready to vacuum this time after the roomba was stopped with the home app.

**Comment by DaveLinger** [comment]
Room 1: Guest Bedroom
Room 6: Living Room
Room 25: Guest Bathroom
Room 26: Master Bathroom
Room 27: Master Bedroom
Room 28: Kitchen
Room 29: Laundry Room
Room 30: Bathroom
Room 31: Entryway
Room 32: Office
Room 33: Hallway
Room 34: Closet

**Comment by bloomkd46** [comment]
> Weird behavior when I tried to stop/dock roomba:
> 
> `Set Roomba To stop and dock With args:undefined`
> 
> It stopped but didn't dock. After a minute, I checked the irobot app. It said saving map. After another minute, roomba started again, then stopped again. I tapped return home in the irobot app and it went home.

Try changing your config to pause and dock. I recall having this same problem with my i8
